Library/Homebrew/test/formula_spec.rb

```diff
@@ -699,6 +699,27 @@
     end
   end
 
+  specify "#service" do
+    f = formula do
+      url "https://brew.sh/test-1.0.tbz"
+      service do
+        arguments ["/usr/local/bin/beanstalkd"]
```

Comment:
Maybe `run` or something instead? I'm not convinced the plist names are always the best.